### ISSUE TITLE
Add documentation for ÖV Güteklassen

### ISSUE THREAD

**Issue by EPajares** [goat-community/goat#1536]
# Goal of this issue
The would need documentation for ÖV Güteklassen on the website. 

# Resources

# Deliverables

# Branch to derive

**Comment by UJehle** [comment]
Task:

- [ ] translation into German

**Comment by UJehle** [comment]
@EPajares I translated it to English. Therewith, this task should be finished? Or do you think we should add something else to this section?

**Comment by EPajares** [comment]
Overall, I believe it is fine. Maybe you could double check some things:

- radii? rather buffer?
- we could discuss wether it makes sense to translate ÖV Güteklassen? Maybe we keep the German wording?
- partly American and British English is mixed (e.g. institutionalisation)
- 6-20 hrs -> 6 AM - 8 AM
- underground ==> metro like in the table

**Comment by UJehle** [comment]
Thanks for the comments! I fixed them. 
Regarding the German term "ÖV Güteklassen" I am not sure. When I used this term in Ghent everybody was quite confused. But when using "Public transport quality classes" they at least had a rough idea of what I am talking about. A quick verbal survey in the Melbourne office had the same results. 
Anyhow, I now use both of them in the heading and in the text _ÖV Güteklassen_ after introducing what it means. I hope that it is understandable for everybody like this.
